library/scss/helpers/_reset.scss

```diff
@@ -0,0 +1,7 @@
+.hustle-ui {
+
+	* ,
+    &.hustle-ui{
```

Comment:
@creador-dev this part doesn't make sense, mate. You sure you want to add styles for a `div` that has `hustle-ui` class name twice?

Reply:
@iamleigh I want to apply box-sizing to hustle-ui as well as made a change will this work out?